fixtures/test-supplier.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
As mentioned above, possible don't needs this with the current supplier creation function.